app/models/grda_warehouse/hud/client.rb

```diff
@@ -1655,11 +1655,15 @@ def cas_pregnancy_status
           (m.DueDate.present? && m.DueDate > Date.current - 3.months)
         )
       end.present?
-      vispdat_pregnancy = vispdats.completed.where(pregnant_answer: 1, submitted_at: in_last_year).exists?
-      eto_pregnancy = source_hmis_forms.vispdat.
-        vispdat_pregnant.
-        where(collected_at: in_last_year).
-        exists?
+      vispdat_pregnancy = nil
+      eto_pregnancy = nil
+      unless cas_calculator_instance.unrelated_columns.include?(:vispdat_score)
+        vispdat_pregnancy = vispdats.completed.where(pregnant_answer: 1, submitted_at: in_last_year).exists?
+        eto_pregnancy = source_hmis_forms.vispdat.
+          vispdat_pregnant.
+          where(collected_at: in_last_year).
+          exists?
+      end
 
       hmis_pregnancy || vispdat_pregnancy || eto_pregnancy
```

Comment:
Not sure if it's an issue, but this previously only returned True or False and now it can return Nil. Maybe the default values for `vispdat_pregnancy` and `eto_pregnancy` should be false instead.

Reply:
I debated.  I'll switch to `false`